src/lug/System/Unix/PathImplUnix.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <lug/System/Path.hpp>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <pwd.h>
+
+namespace lug {
+namespace System {
+namespace priv {
+class PathImpl {

```

Comment:
Add newlines around the class definition, (to separate from the namespaces) and before each `public` or `private`